### DISCUSSION TITLE
Can anyone help me fix this please?

### DISCUSSION THREAD

**Discussion by macerg** [termux/termux-packages#18731] [Q&A] [answered]
`pkg install libsecret
Checking availability of current mirror:
[*] https://linux.domainesia.com/applications/termux/termux-main: ok
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following NEW packages will be installed:
  libsecret
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
8 not fully installed or removed.
Need to get 0 B/187 kB of archives.
After this operation, 1458 kB of additional disk space will be used.
(Reading database ... 68829 files and directories currently installed.)
Preparing to unpack .../libsecret_0.21.2_aarch64.deb ...
Unpacking libsecret (0.21.2) ...
dpkg: error processing archive /data/data/com.termux/cache/apt/archives/libsecret_0.21.2_aarch64.deb (--unpack):
 trying to overwrite '/data/data/com.termux/files/usr/opt/glib/cross/share/glib-2.0/codegen/__pycache__/config.cpython-310.pyc', which is also in package pipewire 1.0.0
dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)
Errors were encountered while processing:
 /data/data/com.termux/cache/apt/archives/libsecret_0.21.2_aarch64.deb
E: Sub-process /data/data/com.termux/files/usr/bin/dpkg returned an error code (1)`

Help much appreciated!

**Comment by twaik** [comment]
First of all `pkg up`...

**Reply by macerg** [reply]
Already done that.

**Comment by macerg** [comment]
`pkg upgrade
Checking availability of current mirror:
[*] https://linux.domainesia.com/applications/termux/termux-main: ok
Hit:1 https://linux.domainesia.com/applications/termux/termux-main stable InRelease
Hit:2 https://linux.domainesia.com/applications/termux/termux-root root InRelease
Hit:3 https://linux.domainesia.com/applications/termux/termux-x11 x11 InRelease
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
You might want to run 'apt --fix-broken install' to correct these.
The following packages have unmet dependencies:
 vlc-qt : Depends: libsecret but it is not installed
E: Unmet dependencies. Try 'apt --fix-broken install' with no packages (or specify a solution).`

**Comment by macerg** [comment]
Tried the `apt --fix-broken install` also

**Comment by licy183** [comment, marked as answer]
Please do not report bugs in discussion...

The latest version of libsecret is `0.21.2-1`, and it has removed the conflict files. This disscussion/issue can be closed now.

**Reply by macerg** [reply]
I wasn't sure that it was a bug as such and didn't want to waste developer's time. I was trying to be considerate. Sorry, straight to bug report next time.
